src/utils/color.rs

```diff
@@ -0,0 +1,52 @@
+use ansi_term;
+use ansi_term::Colour::{Cyan, Green, Red, Yellow};
+
+pub enum Style {
+    Link,          // URLs and graph refs
+    Command,       // Commands, inline code, env variable keys, and profile names
+    Path,          // File paths
+    HintPrefix,    // "HINT:" text
+    WarningPrefix, // "WARN:" text
+    ErrorPrefix,   // "ERROR:", "error:", and "error[code]:" text
+    Heading,
+    CallToAction,
+    WhoAmIKey,
+}
```

Comment:
These aren't exactly the same as what was proposed in https://github.com/apollographql/rover/issues/309, but are easy to change. In general I leaned towards introducing more variants, since it's a lot easier to combine variants than split them apart (the latter requires understanding the intent of calling code).

Reply:
Graph refs were one of the more frequently colored data and may warrant a separate variant from `Link`.